### ISSUE TITLE
Codesnippet feedback attribute needs a corresponding aria-label

### ISSUE THREAD

**Issue by lonestor** [carbon-design-system/carbon#6422]
## Environment

> macOs

> chrome

> Automated testing tool and ruleset

> Assistive technology used to verify: DAP tool

## Detailed description

> What version of the Carbon Design System are you using? carbon-components react v7.15.0

> What did you expect to happen? To be provided an feedback text aria-label

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?
"Accessible name does not contain the visible text label"

## Steps to reproduce the issue

1. Step one 
Add multi CodeSnippet with copy button
2. Step two
Run DAP tool
3. Step three
See violations in the dev console
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox
<img width="607" alt="Screenshot 2020-07-09 at 09 54 19" src="https://user-images.githubusercontent.com/68054577/87020739-2f57e500-c1cc-11ea-878f-1b1767a82753.png">

## Additional information

- Screenshots or code
- Notes
